std/datetime/timezone.d

```diff
@@ -1994,41 +1994,37 @@ public:
     }
 
 
-    version(TZDatabaseDir)
+    version(StdDdoc)
     {
-        import std.string : strip;
         /++
-            The default directory where the TZ Database files are. It's empty
-            for Windows, since Windows doesn't have them.
+            The default directory where the TZ Database files are stored. It's
+            empty for Windows, since Windows doesn't have them. You can also use
+            the TZDatabaseDir version to pass an arbitrary path at compile-time,
+            rather than hard-coding it here. Android concatenates all time zone
+            data into a single file called tzdata and stores it in the directory
+            below.
           +/
+        enum defaultTZDatabaseDir = "";
```

Comment:
First `StdDdoc` comment I've written, not sure if I need to put this blank enum here.

Reply:
You do. You can't document a symbol that isn't there.